.cruft.json

```diff
@@ -0,0 +1,43 @@
+{
+    "template": "https://github.com/scverse/cookiecutter-scverse",
+    "commit": "d383d94fadff9e4e6fdb59d77c68cb900d7cedec",
+    "checkout": "v0.6.0",
+    "context": {
+        "cookiecutter": {
+            "project_name": "anndata",
+            "package_name": "anndata",
+            "project_description": "Annotated data.",
+            "author_full_name": "Philipp Angerer",
+            "author_email": "philipp.angerer@helmholtz-munich.de",
+            "github_user": "scverse",
+            "github_repo": "anndata",
+            "license": "BSD 3-Clause License",
+            "ide_integration": true,
+            "_copy_without_render": [
+                ".github/workflows/build.yaml",
+                ".github/workflows/test.yaml",
```

Comment:
Wouldn't we want to change the name of our files to get updates from these? I think our tests now are called `test-cpu.yml`. As far as I understand it, this section is not used after the initial directory creation.  Still my question stands.

Reply:
copy without render means that cookiecutter won't do cookiecutter variable interpolation for these files.

You'll get updates for these files in any case. What will happen is that it'll try to add these files in addition to your existing ones. You can sort it out manually then or rename your existing one to match the template names.